util/options_builder.cc

```diff
@@ -7,6 +7,8 @@
 #include <algorithm>
 #include "rocksdb/options.h"
 
+using std::log;
```

Comment:
We follow Google C++ style, which forbids using namespace: https://google.github.io/styleguide/cppguide.html#Namespaces "Named Namespaces" section.

Is there a way you can spell out where the namespaces are used?

Same comment for utilities/geodb/geodb_impl.cc.